package.json

```diff
@@ -15,7 +15,7 @@
     "build": "yarn clean && yarn rollup -c",
     "clean": "rimraf dist",
     "ci": "yarn lint && yarn test && yarn test:versions && yarn test:types && yarn typecheck && yarn build",
-    "prepublish": "yarn ci",
+    "prepublishOnly": "yarn ci",
```

Comment:
`prepack` ?

Reply:
I think either works, is there a reason to prefer one or the other?

Reply:
Some versions of NPM ran `prepublishOnly` after pack 🤦‍♂ 
https://github.com/npm/npm/issues/15454